_commands/scripting/dollarUnderscore.md

```diff
@@ -0,0 +1,41 @@
+---
+---
+
+$\_
+-----
+
+`$_` is a special shell variable which always holds the last argument of the most recent command.
+
+~~~ bash
+$ touch example.txt
```

Comment:
Add an example that has more than two words in it so that it's clear which word it's selecting.